samples/bluetooth/tmap_peripheral/src/tmap_peripheral.h

```diff
@@ -4,7 +4,9 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-#include <zephyr/types.h>
+#include <stdint.h>
```

Comment:
Arguably and for consistency reasons a copyright line should be added

Reply:
Did consider it, but figured that the size of the change here did not really warrant that.